crates/common/validator/src/contribution_and_proof.rs

```diff
@@ -29,3 +37,38 @@ pub struct SignedContributionAndProof {
     pub message: ContributionAndProof,
     pub signature: BLSSignature,
 }
+
+pub fn get_contribution_and_proof(
+    state: &BeaconState,
+    aggregator_index: u64,
+    contribution: SyncCommitteeContribution,
+    private_key: PrivateKey,
+) -> anyhow::Result<ContributionAndProof> {
+    let slot = contribution.slot;
+    let subcommittee_index = contribution.slot;
+    Ok(ContributionAndProof {
+        aggregator_index,
+        contribution,
+        selection_proof: get_sync_committee_selection_proof(
+            state,
+            slot,
+            subcommittee_index,
+            private_key,
+        )?,
+    })
+}
```

Comment:
```suggestion
     let selection_proof = get_sync_committee_selection_proof(
        state,
        contribution.slot,
        contribution.subcommittee_index,
        private_key,
    )?;
    Ok(ContributionAndProof {
        aggregator_index,
        contribution,
        selection_proof,
    })
```

We should subcommittee_index properly

Reply:
`let subcommittee_index = contribution.slot;`

`subcommittee_index` was a slot literally